physics/mass_energy_equivalence.py

```diff
@@ -0,0 +1,78 @@
+"""
+Title:
+Finding the energy equivalence of mass and mass equivalence of energy
+by Einstein's equation.
+
+Description:
+Einstein's mass-energy equivalence is a pivotal concept in theoretical physics.
+It asserts that energy (E) and mass (m) are directly related by the speed of
+light in vacuum (c) squared, as described in the equation E = mc². This means that
+mass and energy are interchangeable; a mass increase corresponds to an energy increase,
+and vice versa. This principle has profound implications in nuclear reactions,
+explaining the release of immense energy from minuscule changes in atomic nuclei.
+
+Equations:
+E = mc² and m = E/c², where m is mass, E is Energy, c is speed of light in vacuum.
+
+Reference:
+https://en.wikipedia.org/wiki/Mass%E2%80%93energy_equivalence
+"""
+
+# speed of light in vacuum (c) in m/s
+SPEED_OF_LIGHT = 299792458.0
```

Comment:
```suggestion
from scipy.constants import c  # speed of light in vacuum (299792458 m/s)
```
Let's just import the constant from `scipy` because that leaves one fewer place for potential errors.